docs/pages/attributesJSON.rst

```diff
@@ -177,6 +174,12 @@ Below are object-specific physical quantities.  These values will override simil
 "requires_lighting"
 	- boolean
 	- Whether the object should be rendered with lighting or flat shading.
+"is_visible"
+	- boolean
+	- Whether the object's render asset should be created upon instancing.
+"is_collidable"
```

Comment:
If it's not collidable, does that also imply no dynamics? Not affected by gravity?

Reply:
Thanks for the comment here. I think re-naming this field to something more semantically accurate makes sense. Maybe "is_physical", or even "is_visualization_only".

Reply:
@eundersander @aclegg3  There has to be a term from this we can borrow from game_design, right?

Reply:
* incorporeal - having no physical body or form 
* ethereal - lack material substance
* is_ghost - ghosts can pass through things. ;)